eng/scripts/Language-Settings.ps1

```diff
@@ -0,0 +1,42 @@
+$Language = "go"
+$LanguageDisplayName = "go"
+
+function Get-AllPackageInfoFromRepo ($serviceDirectory)
+{
+    $allPackageProps = @()
+    $searchPath = Join-Path $RepoRoot "sdk"
+    if ($serviceDirectory)
+    {
+        $searchPath = Join-Path $searchPath $serviceDirectory 
+    }
+
+    $pkgFiles = Get-ChildItem -Path $searchPath -Include "go.mod" -Recurse
+
+    foreach ($pkgFile in $pkgFiles)
+    {
+        $pkgPath = $pkgFile.DirectoryName
+        $pkgName = $pkgFile.Directory.Name
+        $pkgVersion = ""
+        $serviceDirectory = $pkgName
+
+        if ($pkgPath -match ".*\\sdk\\(?<serviceDir>.*)")
+        {
+            $serviceDirectory = $Matches["serviceDir"]
+        }
+        $versionFile = Join-Path $pkgPath "version.go"
```

Comment:
This line is assuming that every package in this repo has a `version.go` file which contains the version number. This might be correct for dataplane packages, but unfortunately we do not have this file in mgmt plane packages.
Should we change the generator so that we also generate a `version.go` or change the logic in the generator so that we could automatically apply the version number to `version.go`? @jhendrixMSFT @chamons